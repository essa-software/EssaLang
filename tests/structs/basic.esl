// output: 2137 8888888

struct Test {
    your: u32;
    mum: u32;
}

func main() {
    mut test: Test = Test { your: 2137, mum: 111 };
    test.mum = 8888888;
    print("{} {}\n", test.your, test.mum);
}
