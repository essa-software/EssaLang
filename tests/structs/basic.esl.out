2137 8888888
